Formula/jenkins-lts.rb

```diff
@@ -1,16 +1,16 @@
 class JenkinsLts < Formula
   desc "Extendable open-source CI server"
   homepage "https://jenkins.io/index.html#stable"
-  url "http://mirrors.jenkins-ci.org/war-stable/2.73.2/jenkins.war"
-  sha256 "f6d1351beef34d980b32f8c463be505445f637e2fc62156fecd42891c53c97d3"
+  url "http://mirrors.jenkins.io/war-stable/2.73.3/jenkins.war"
+  sha256 "fcfb932a84ce03795abbf51ae7799439278155b0d42242d1d79a187034649fc8"
 
   bottle :unneeded
 
   depends_on :java => "1.7+"
 
   def install
     system "jar", "xvf", "jenkins.war"
-    libexec.install Dir["jenkins.war", "WEB-INF/jenkins-cli.jar"]
+    libexec.install %w[jenkins.war WEB-INF/jenkins-cli.jar]
```

Comment:
The correct fix here is `libexec.install "jenkins.war", "WEB-INF/jenkins-cli.jar"`

Reply:
Sure, will change. Wasn't sure which format was the most desired here these days to be honest.